treeherder/model/migrations/0021_add_jobdetail_model.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import treeherder.model.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0020_update_job_name_length'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='JobDetail',
+            fields=[
+                ('id', treeherder.model.fields.BigAutoField(serialize=False, primary_key=True)),
+                ('title', models.CharField(max_length=512, null=True)),
```

Comment:
These changes won't get applied on stage automatically, since the migration with the same name will have already run as part of the prior deployment - but that had a max_length of 200 characters (or null in the case of `url`). Depending on what the auto-deploy settings are for the heroku instances, it may also have been applied there.

I think maybe the best way to revert changes like this is to land schema changes in a separate commit, and if reverting the PR, to either not revert that commit at all, or to 'revert' the change by creating a new migration that undoes the original migration.